manifests/d/Dagger/Cli/0.9.6/Dagger.Cli.locale.en-US.yaml

```
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: Dagger.Cli
```

```suggestion
PackageIdentifier: Dagger.DaggerCli
```
- Please remember to update the file path and file name to match this change.